### ISSUE TITLE
No focus indicator on Radio button in Windows High Contrast mode

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#767]
### Expected Behaviour
Should be able to see the keyboard focus

### Actual Behaviour
Cannot see the keyboard focus

### Reproduce Scenario (including but not limited to)
https://opensource.adobe.com/spectrum-web-components/components/radio/examples

#### Steps to Reproduce
Enable Windows High Contrast mode
Tab through the page and note that you cannot discern the focus

#### Platform and Version
Windows 10
Edge Chromium

**Comment by Westbrook** [comment]
Is this still related to #297 in that this originates upstream? Or have you seen motion of [the issue therein](https://github.com/adobe/spectrum-css/issues/168) that we've yet to adopt?
